packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts

```diff
@@ -0,0 +1,542 @@
+                    MULTIDB_SUPPORT = false;
+                } else {
+                    throw e;
+                }
+            }
+        } finally {
+            await session.close();
+        }
+
+        await delay(5000);
+    });
+
+    afterAll(async () => {
+        if (MULTIDB_SUPPORT) {
+            const cypher = `DROP DATABASE ${databaseName}`;
+
+            const session = await neo4j.getSession();
+            try {
+                await session.run(cypher);
+            } finally {
+                await session.close();
+            }
+        }
+
+        await driver.close();
+    });
+
+    testIf(MULTIDB_SUPPORT)("should create index if it doesn't exist", async () => {
+        const title = generate({ readable: true, charset: "alphabetic" });
+        const indexName = generate({ readable: true, charset: "alphabetic" });
+        const type = new UniqueType("Movie");
+
+        const typeDefs = gql`
+            type ${type.name} @fulltext(indexes: [{ indexName: "${indexName}", fields: ["title"] }]) {
+                title: String!
+            }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        await neoSchema.getSchema();
+        await expect(
+            neoSchema.assertIndexesAndConstraints({
+                driver,
+                driverConfig: { database: databaseName },
+                options: { create: true },
+            })
+        ).resolves.not.toThrow();
+
+        const session = driver.session({ database: databaseName });
+
+        const cypher = `
+            SHOW INDEXES yield
+                name AS name,
+                type AS type,
+                entityType AS entityType,
+                labelsOrTypes AS labelsOrTypes,
+                properties AS properties
+            WHERE name = "${indexName}"
+            RETURN {
+                 name: name,
+                 type: type,
+                 entityType: entityType,
+                 labelsOrTypes: labelsOrTypes,
+                 properties: properties
+            } as result
+        `;
+
+        try {
+            const result = await session.run(cypher);
+
+            const record = result.records[0].get("result") as {
+                name: string;
+                type: string;
+                entityType: string;
+                labelsOrTypes: string[];
+                properties: string[];
+            };
+
+            expect(record.name).toEqual(indexName);
+            expect(record.type).toBe("FULLTEXT");
+            expect(record.entityType).toBe("NODE");
+            expect(record.labelsOrTypes).toEqual([type.name]);
+            expect(record.properties).toEqual(["title"]);
+
+            await session.run(`
+                CREATE (:${type.name} { title: "${title}" })
+            `);
+        } finally {
+            await session.close();
+        }
+    });
+
+    testIf(MULTIDB_SUPPORT)("should create index if it doesn't exist (using node label)", async () => {
```

Comment:
Is there a case for multiple node labels?

Reply:
I... think we don't support this case, I'll add a card